Trimble.Modus.Components/Controls/TMCard/TMCard.xaml.cs

```diff
@@ -0,0 +1,136 @@
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;
+using Microsoft.Maui.Controls.Shapes;
+using System.Windows.Input;
+
+namespace Trimble.Modus.Components.Controls.TMCard;
+
+public partial class TMCard : ContentView
+{
+    #region Fields 
+    private readonly TapGestureRecognizer _tapGestureRecognizer;
+    private Shadow _shadow;
+    private Border _border;
+    private EventHandler _clicked;
+    private const int _borderRadius = 2;
+    #endregion
+
+    #region Public properties
+
+
+    public bool IsSelected
+    {
+        get { return (bool)GetValue(IsSelectedProperty); }
+        set { SetValue(IsSelectedProperty, value); }
+    }
+
+    public event EventHandler Clicked
+    {
+        add { _clicked += value; }
+        remove { _clicked -= value; }
+    }
+    public ICommand Command
+    {
+        get => (Command)GetValue(CommandProperty);
+        set => SetValue(CommandProperty, value);
+    }
+    public object CommandParameter
+    {
+        get => GetValue(CommandParameterProperty);
+        set => SetValue(CommandParameterProperty, value);
+    }
+    #endregion
+
+    #region Bindable Properties
+
+
+    public static readonly BindableProperty IsSelectedProperty =
+        BindableProperty.Create(nameof(IsSelected), typeof(bool), typeof(TMCard), defaultValue: false, propertyChanged: isSelectedChanged);
+
+    public static readonly BindableProperty ClickedEventProperty =
+          BindableProperty.Create(nameof(Clicked), typeof(EventHandler), typeof(TMCard));
+
+    public static readonly BindableProperty CommandProperty =
+           BindableProperty.Create(nameof(Command), typeof(Command), typeof(TMCard), null);
+
+    public static readonly BindableProperty CommandParameterProperty =
+        BindableProperty.Create(nameof(CommandParameter), typeof(object), typeof(TMCard), null);
+    #endregion
+
+    #region Property Changes
+
+    private static void isSelectedChanged(BindableObject bindable, object oldValue, object newValue)
+    {
+        var card = (TMCard)bindable;
+        if ((bool)newValue)
+        {
+            card._border.BackgroundColor = (Color)BaseComponent.colorsDictionary()["ToastBlue"];
+            card._border.Stroke = (Color)BaseComponent.colorsDictionary()["TrimbleBlueClicked"];
+        }
+        else
+        {
+            card._border.BackgroundColor = Colors.White;
+            card._border.Stroke = Colors.Transparent;
+        }
+    }
+
+    #endregion
+
+    public TMCard()
+    {
+        _shadow = new Shadow
```

Comment:
The shadow is not even showing in Android 12 simulator that I use. And it is too big in iOS. @BAL4-R Can you check this iOS and see if the shadow and border is compliant with the figma?
![Simulator Screen Shot - iPhone 14 - 2023-06-02 at 12 56 04](https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/01da0593-af54-4f4e-86df-2df54246902d)

Reply:
Seems like shadow radius is getting rendered differently in Android, iOS and Windows. 
https://github.com/dotnet/maui/issues/8796#issuecomment-1514716353

Ill use platform specific radius for this

Reply:
@rthanga1 The shadow is not visible in simulator but its visible in real device.

**Windows**
![Windows](https://github.com/trimble-oss/modus-mobile-maui-components/assets/130034598/be797df7-403a-4cb3-bb5a-846585972e53)

**iOS**
![ios](https://github.com/trimble-oss/modus-mobile-maui-components/assets/130034598/6f3cdd76-6647-4787-b14a-6d8abda8169b)

**Android**
![Android](https://github.com/trimble-oss/modus-mobile-maui-components/assets/130034598/76d052e9-af37-4e8e-9336-350b4e29d3dd)